Editor/Migration/Migrator.cs

```diff
@@ -195,15 +195,15 @@ private static bool DoMigrate()
             InProgress = true;
             var result = EditorUtility.DisplayDialog("Migrate!",
```

Comment:
ダウングレード検出時のダイアログのタイトルは日本語なので、マイグレーション時も日本語で良いのではという感じもあります。
```suggestion
            var result = EditorUtility.DisplayDialog("マイグレーションが必要です!",
```
みたいな…